src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,190 @@
+  }
+
+  private void verifyResponse(InventoryItemResource smallAngryPlanet,
+                              UUID secondServicePointId,
+                              List<JsonObject> items) {
+    JsonObject itemJson = items.get(0);
+    assertThat(itemJson.getString(BARCODE_KEY), is(smallAngryPlanet.getBarcode()));
+    assertThat(itemJson.getJsonObject(STATUS_KEY).getMap().get("name"),
+      is(ItemStatus.IN_TRANSIT.getValue()));
+    assertThat(itemJson.getString(DESTINATION_SERVICE_POINT), is(String.valueOf(secondServicePointId)));
+    final JsonObject smallAngryPlanetInstance = smallAngryPlanet.getInstance().getJson();
+    assertThat(itemJson.getString(TITLE), is(smallAngryPlanetInstance.getString(TITLE)));
+    final String contributors = String.valueOf(((JsonArray) smallAngryPlanetInstance
+      .getMap().get(CONTRIBUTORS)).getJsonObject(0).getMap().get("name"));
+    assertThat(itemJson.getJsonArray(CONTRIBUTORS)
+      .getJsonObject(0).getMap().get("name"), is(contributors));
+    Map<String, String> actualLocation = (Map<String, String>) itemJson.getMap().get("location");
+    assertThat(actualLocation.get(LOCATION_NAME), is("3rd Floor"));
+    assertThat(actualLocation.get(LOCATION_CODE), is("NU/JC/DL/3F"));
+    assertThat(actualLocation.get(LIBRARY), is("Djanogly Learning Resource Centre"));
+  }
+
+  @Test
+  public void reportIncludeMultipleDifferentItemsInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+    createRequest(nod, rebecca, secondServicePointId);
+
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+    loansFixture.checkInByBarcode(nod);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertThat(items.size(), is(2));
+  }
+
+  @Test
+  public void reportExcludesItemsOtherThenInTransitStatus()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
```

Comment:
This test could be working because the items are filtered by whether there is a request present or not. Personally, I would include an item with a request that was in transit as well.

Reply:
Request fields will be added in another story. I think it is more reasonable to add this test in scope of https://issues.folio.org/browse/CIRC-519